eng/utils.ps1

```diff
@@ -156,56 +65,22 @@ function Use-LlvmInstallation {
     $env:LLVM_SYS_110_PREFIX = $path
 }
 
-# Gets the LLVM version git hash or tag name
-function Get-LlvmTag {
-    "llvmorg-11.1.0"
-}
-
-function Get-PackageName {
-    $tag = Get-LlvmTag
-    $TARGET_TRIPLE = Get-TargetTriple
-    $packageName = "qirlib-llvm-$($TARGET_TRIPLE)-$($tag)"
-    $packageName
-}
-
-function Get-DefaultInstallDirectory {
-    if (Test-Path env:\PYQIR_CACHE_DIR) {
-        $env:PYQIR_CACHE_DIR
+function Resolve-InstallationDirectory {
+    if (Test-Path env:\QIRLIB_LLVM_EXTERNAL_DIR) {
+        return $env:QIRLIB_LLVM_EXTERNAL_DIR
     }
     else {
-        Join-Path "$HOME" ".pyqir"
-    }
-}
-
-function Get-AqCacheDirectory {
-    $aqCacheDirectory = (Get-DefaultInstallDirectory)
-    if (!(Test-Path $aqCacheDirectory)) {
-        mkdir $aqCacheDirectory | Out-Null
+        $packagePath = Get-DefaultInstallDirectory
+        return $packagePath
     }
-    Resolve-Path $aqCacheDirectory
-}
-
-function Get-InstallationDirectory {
-    [CmdletBinding()]
-    param (
-        [Parameter()]
-        [string]
-        $packageName
-    )
-    $aqCacheDirectory = Get-AqCacheDirectory
-    $packagePath = Join-Path $aqCacheDirectory $packageName
-    $packagePath
 }
 
-function Resolve-InstallationDirectory {
-    if (Test-Path env:\PYQIR_LLVM_EXTERNAL_DIR) {
-        return $env:PYQIR_LLVM_EXTERNAL_DIR
+function Get-DefaultInstallDirectory {
+    if (Test-Path env:\QIRLIB_CACHE_DIR) {
+        $env:QIRLIB_CACHE_DIR
     }
```

Comment:
I assume this is equivalent to 
```suggestion
function Get-DefaultInstallDirectory {
    if (Test-Path env:\QIRLIB_CACHE_DIR) {
        return $env:QIRLIB_CACHE_DIR
    }
```
but I'm not sure if it works in .ps1 (works in .rs).

Reply:
Both the if and else blocks have implicit returns